### ISSUE TITLE
fx128: Ubuntu 24.04 transparent line between the menubar and tabbar

### ISSUE THREAD

**Issue by adomasven** [zotero/zotero#4922]
Default Ubuntu 24.04 configuration with Wayland and Gnome.

![Screenshot from 2024-12-11 12-03-14](https://github.com/user-attachments/assets/01ee37e3-3558-4614-af6e-e53827b3d2ef)

The line is transparent and shows whatever is behind the window.

**Comment by abaevbog** [comment]
For some reason, github didn't close the issue even though I added "Fixes: " to the commit and explicitly linked the PR... Odd. But that margin tweak to push the titlebar a bit down should close that gap.

**Comment by dstillman** [comment]
It only auto-closes for commits on the main branch.

**Comment by abaevbog** [comment]
Ah! Thank you, good to know.

**Comment by dstillman** [comment]
This isn't right, though. The line is still more prominent than it was before.

Z7:

![Screenshot 2024-12-13 at 5 18 58 AM](https://github.com/user-attachments/assets/a3b24196-d354-4e86-8934-57c6aaf83e01)

fx128

![Screenshot 2024-12-13 at 5 19 41 AM](https://github.com/user-attachments/assets/8c330012-a759-45df-8938-5372efbba3db)

**Comment by abaevbog** [comment]
It looks like the color of the border `--toolbar-field-border-color` is actually different now from what it was pre-fx128. It used to be `ThreeDShadow` but now it is `--input-border-color`. To me, `ThreeDShadow` now looks a bit off and different from what it was but `transparent` border looks about right. I added a PR with that tweak. There are also screenshots as to what the border would look like if we just set it to be `ThreeDShadow` again, correct me if that one actually looks better.
